apps/dash-tsne/runtime.txt

```diff
@@ -1 +1,2 @@
-python-3.6.6
\ No newline at end of file
+python-3.6.6
```

Comment:
Can you remove the newline here? This diff isn't super meaningful, so it shouldn't be in the PR.